audio/filter/af_rubberband.c

```diff
@@ -87,6 +97,19 @@ static int control(struct af_instance *af, int cmd, void *arg)
         p->pending = NULL;
         p->rubber_delay = 0;
         return AF_OK;
+    case AF_CONTROL_COMMAND: {
+        char **args = arg;
+        if (!strcmp(args[0], "set-pitch")) {
+            char *endptr;
+            double pitch = strtod(args[1], &endptr);
```

Comment:
That's... a bug in the parser for af_pan, I think (which I didn't catch). The intention was `while(*cp`... instead, checking for a terminator, not the pointer itself. Though due to the code logic I don't think it actually triggers any badness if cp ends prematurely. I can fix that in another change.

Other than that, no, I don't think args[1] can ever be NULL. The logic in player/command.c seems to always ensure that it isn't. af_lavfi doesn't check it either.